plugins/module_utils/mso.py

```diff
@@ -1553,6 +1553,21 @@ def input_validation(self, attr_name, attr_value, required_attributes, target_ob
         elif empty_attributes:
             self.module.fail_json(msg="When the '{0}' is '{1}', the {2} attributes must be set".format(attr_name, attr_value, empty_attributes))
 
+    def get_template_object_by_uuid(self, object_type, uuid, fail_module=True):
```

Comment:
I would move this to utils file

Reply:
Moved this to the MSOModule class for a few reasons.
1. It already requires us to pass in the MSOModule object if we add it to `utils.py`. This reduces the number of required arguments in the function.
2. The function is already available via the mso object so there is no need to import it from utils.
Please let me know if there is a reason to keep it in utils.py instead.